code/game/objects/items/toys.dm

```diff
@@ -589,9 +594,14 @@
 
 /obj/item/toy/plushie/attack_self(mob/user as mob)
 	if(has_stuffing || grenade)
-		var/cuddle_verb = pick("hugs", "cuddles", "snugs")
-		user.visible_message("<span class='notice'>[user] [cuddle_verb] [src].</span>")
-		playsound(get_turf(src), poof_sound, 50, TRUE, -1)
+		if(rare_hug_sound && rare_hug_word && COOLDOWN_FINISHED(src, rare_hug_cooldown))
```

Comment:
I think you might want to split up this condition so you can still get the rare_hug_sound even if there isn't a hug word, and vice versa.

Reply:
They typically only come together, I think the code should not run (or fail) when theres not both